src/oogeso/dto/oogeso_output_data_objects.py

```diff
@@ -32,7 +32,7 @@ class SimulationResult:
     # Max available "flow" (power/fluid) from storage (Sm3/s or MW):
     dfDeviceStoragePmax: pd.Series = None
     # Device assosiated penalty rate (PENALTY_UNIT/s):
-    dfPenalty: pd.DataFrame = None
+    dfPenalty: pd.Series = None
```

Comment:
It is sufficient to do dfPenalty: pd.Series unless you want it optional. Then you should do df: Optional[pd.Series] = None

Reply:
OK, removed default None value